Mail/AppDelegate.swift

```diff
@@ -106,12 +106,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let networkLoginService = Factory(type: InfomaniakNetworkLogin.self) { _, _ in
```

Comment:
This is a nice to have, but while injecting objects, we might want to register `InfomaniakNetworkLoginable.self` here.
That way it will be easier to test.